### DISCUSSION TITLE
The option --no-taskbar-progress doesn't seem to work

### DISCUSSION THREAD

**Discussion by superpoincare** [mpvnet-player/mpv.net#470] [Q&A]
--no-taskbar-progress in mpv.conf doesn't seem to work. 

Something I am doing wrong?

**Comment by hooke007** [comment]
https://github.com/stax76/mpv.net/blob/master/docs/Manual.md#window-limitations

No implement so far.

**Reply by stax76** [reply]
It's implemented but reads the value directly from the conf file instead of querying the mpv API, it should work using `taskbar-progress = no`.
